File: Java/catAndMouse.java
import java.lang.Math;

class catAndMouse{
    public static void main(String[] args) {
        int catM[] = { 1, 3, 2 };
        System.out.print(catMouse(catM[0], catM[1], catM[2]));
    }

    public static String catMouse(int cat1, int cat2, int mouse ) {
       if (Math.abs(cat1 - mouse) == Math.abs(cat2 - mouse)) {
            return "Mouse C";
        } else if(java.lang.Math.abs(cat1 - mouse) < java.lang.Math.abs(cat2 - mouse)) {
            return "Cat A";
        } else {
            return "Cat B";
        }
    }
}

File: Python/catandmouse.py
def catMouse(cat1, cat2, mouse):
    if abs(cat1 - mouse) == abs(cat2 - mouse):
        return "Mouse C"
    elif abs(cat1 - mouse) < abs(cat2 - mouse):
        return "Cat A"
    else:
        return "Cat B"

print(catMouse(1, 2, 3))